src/test/java/io/github/bookcrawler/core/impl/packtparser/PacktBookInfoParserTest.java

```diff
@@ -0,0 +1,47 @@
+package io.github.bookcrawler.core.impl.packtparser;
+
+import io.github.bookcrawler.core.impl.SourceScrappingResult;
+import io.github.bookcrawler.entities.BookInfo;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class PacktBookInfoParserTest {
+
+    @Test
+    public void testParsingBookElementsFromDocument() throws IOException {
+
+        //given
+        PacktBookInfoParser packtBookInfoParser = new PacktBookInfoParser();
+        SourceScrappingResult sourceScrappingResult = mock(SourceScrappingResult.class);
+        Document source = Jsoup.parse(new File("src/test/java/io/github/bookcrawler/core/" +
```

Comment:
In JsoupSourceScrapper you catch Exception and you do nothing with that, in Profile class you have unused import and Long ID break Java setting name concepts. But in overall is very good. For me it can be merged :+1: